strings/src/main/res/values/strings.xml

```diff
@@ -1243,4 +1243,8 @@
 
     <!-- Message above a form draft that has been completed -->
     <string name="complete" oat:toReview="true">Complete</string>
+
+    <!-- Title and message for dialog explaining the drafts marked with "Error" shown to the user the first time they view the Drafts screen -->
+    <string name="drafts_pills_education_title">Drafts have changed</string>
```

Comment:
Should have oat:toReview here! But also not a big deal since we're going to a non-master branch.